api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
+
+def startdb()
```

Comment:
Could you add docstrings (or equivalent)? These are long enough I can't quite scan the function to see quickly what it's doing. Does this just run `mysqld`? Does it also install MySQL if necessary? Does it run db upgrades, or is that in a different step?